src/components/rocm_smi/README.md

```diff
@@ -20,41 +22,48 @@ Typically, the utility `papi_components_avail` (available in `papi/src/utils/pap
 
 ## Environment Variables
 
-For ROCM_SMI, PAPI requires one environment variable: `PAPI_ROCMSMI_ROOT`. Note
-in most installations, this is a subdirectory under the ROCM directory. This is 
-required at both compile and run time.
+For ROCM_SMI, PAPI requires the environment variable `PAPI_ROCMSMI_ROOT` to be set such that the shared object `librocm_smi64.so` and the directory `rocm_smi` are found. This variable is required at both compile and run time.
+
+There are two common cases for setting this variable:
+
+1. **Case 1: For ROCm versions 5.2 and newer:**
+    Set `PAPI_ROCMSMI_ROOT` to the top-level ROCM directory, e.g.:
 
-Example:
+        export PAPI_ROCMSMI_ROOT=/opt/rocm
 
-    export PAPI_ROCMSMI_ROOT=/opt/rocm/rocm_smi
+2. **Case 2: For ROCm versions prior to 5.2:**
+    Set `PAPI_ROCMSMI_ROOT` directly to the ROCM_SMI directory, e.g.:
 
-Within PAPI_ROCMSMI_ROOT, we expect the following standard directories:
+        export PAPI_ROCMSMI_ROOT=/opt/rocm/rocm_smi
```

Comment:
I do not have access to a ROCm version < 5.2. However, I simulated that this would hold with our current paths in `Rules.rocm_smi`. Tested on an MI200 with `papi_component_avail`, `papi_native_avail`, and `papi_command_line` as verification.